### PR TITLE
Added SWAPDB command

### PR DESCRIPTION
**Pull request by cheprasov**

### CODE REVIEW

File: commands/swapdb.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This new command swaps two Redis databases, so that immediately all the
```
Comment (badboy) [inline commands/swapdb.md]: Remove the new. While it is new in 4.0, it won't stay "new"

File: commands/swapdb.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+clients connected to a given DB will see the data of the other DB, and
+the other way around. Example:
```
Comment (badboy) [inline commands/swapdb.md]: Please expand `DB` to `database`. Makes reading easier. Applies to below as well.

**Comment by cheprasov** [comment]
Done.

**Comment by badboy** [comment]
Thanks!

**Pull request merged**